scripts/bygg

```diff
@@ -124,10 +124,21 @@ parse_options()
 # Handles move to templates which Mickledore requires.
 find_templateconf()
 {
-  case "$MACHINE" in
-    imx8mp-var-dart-hmx1) machine_folder="hmx" ;;
-    *) machine_folder="generic" ;;
-  esac
+    case "$MACHINE" in
+        "")
+            >&2 echo "🔴 Error in ${FUNCNAME[0]}: Environment variable MACHINE is not defined."
+            return 1
+            ;;
+        imx8mp-var-dart-hmx1)
+            machine_folder="hmx"
+            ;;
+        verdin-am62-hmm)
+            machine_folder="hmm"
+            ;;
+        *)
+            machine_folder="generic"
+            ;;
+    esac
```

Comment:
Use 2-space indentation as rest of the  script please

Reply:
fixed